diff --git a/lib/location/scan_wifi.c b/lib/location/scan_wifi.c
index 8410ae5bb..ff4dee9db 100644
--- a/lib/location/scan_wifi.c
+++ b/lib/location/scan_wifi.c
@@ -150,7 +150,7 @@ int scan_wifi_cancel(void)
 int scan_wifi_init(void)
 {
 	const struct device *wifi_dev;
-
+#if 0
 	wifi_iface = NULL;
 #if defined(CONFIG_WIFI_NRF700X)
 	wifi_dev = device_get_binding("wlan0");
@@ -171,7 +171,7 @@ int scan_wifi_init(void)
 	net_mgmt_init_event_callback(&scan_wifi_net_mgmt_cb, scan_wifi_net_mgmt_event_handler,
 				     (NET_EVENT_WIFI_SCAN_RESULT | NET_EVENT_WIFI_SCAN_DONE));
 	net_mgmt_add_event_callback(&scan_wifi_net_mgmt_cb);
-
+#endif
 	return 0;
 }
 
